spring-cloud-starter-stream-source-twitterstream/src/main/java/org/springframework/cloud/stream/app/twitterstream/source/TwitterStreamProperties.java

```diff
@@ -37,6 +37,21 @@
 	 */
 	private String language;
 
+	/**
+	 * A comma separated list of user IDs, indicating the users to return statuses for in the stream.
+	 */
+	private String follow;
+
+	/**
+	 * Keywords to track.
+	 */
+	private String track;
+
+	/**
+	 * A set of bounding boxes to track.
+	 */
+	private String locations;
```

Comment:
I suggest to move all these 3 options to the inner `Filter` class. since all of them are related only to the `FILTER` stream type.